pipes/rules/common.rules

```diff
@@ -1,13 +1,26 @@
 
+import os, os.path, sys
 # imports needed for download_file() and webfile_readlines()
 import re
 # since py3 split up urllib
 try:
     from urllib.request import urlopen
 except ImportError:
     from urllib2 import urlopen
+
+# Ensure that Snakemake rules can import viral-ngs python code
+sys.path.append(os.path.realpath(os.path.join(os.path.expanduser(config['bin_dir']))))
```

Comment:
Should we prepend it instead for higher precedence on the path? 

`sys.path.insert(0, <path/to/add>)`